### DISCUSSION TITLE
about SQL assist

### DISCUSSION THREAD

**Discussion by ct-groups** [dbeaver/dbeaver#14671] [General]
how to change the order?
![1637762239(1)](https://user-images.githubusercontent.com/53501828/143251786-2f4a3a26-c485-4a74-8551-d579dca6b6a9.png)

**Comment by LonwoLonwo** [comment]
Hello @rainman-code 

The order of what?

**Comment by ct-groups** [comment]
Display order of content from top to bottom



---Original---
From: ***@***.***&gt;
Date: Wed, Nov 24, 2021 23:11 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [dbeaver/dbeaver] about SQL assist (Discussion #14671)




 
Hello @rainman-code
 
The order of what?
 
—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by ct-groups** [comment]
Sometimes these helper prompts are not as large, presumably because they are not being loaded



---Original---
From: ***@***.***&gt;
Date: Wed, Nov 24, 2021 23:11 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [dbeaver/dbeaver] about SQL assist (Discussion #14671)




 
Hello @rainman-code
 
The order of what?
 
—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by LonwoLonwo** [comment]
Maybe this setting can help you:

![2021-11-24 18_28_45-DBeaver Ultimate 21 3 0 - _postgres_ Script-10](https://user-images.githubusercontent.com/45152336/143267256-1bca7afa-1007-4ec2-90ae-5c690ec8f91b.png)
